utils/frame/benchmarking-cli/src/command.rs

```diff
@@ -126,19 +126,20 @@ impl BenchmarkCmd {
 						// Print the table header
 						batch.results[0].components.iter().for_each(|param| print!("{:?},", param.0));
 
-						print!("extrinsic_time,storage_root_time,reads,repeat_reads,writes,repeat_writes\n");
+						print!("extrinsic_time,storage_root_time,reads,repeat_reads,writes,repeat_writes,proof_size\n");
```

Comment:
```suggestion
						print!("extrinsic_time,storage_root_time,reads,repeat_reads,writes,repeat_writes,proof_size_bytes\n");
```

better to make the name self-descriptive.

Reply:
the time is in milliseconds?

Reply:
def not in milliseconds. nanoseconds it is...